[IMP] events: added supplemental content to create events (#9523)
odoo/documentation: content/applications/marketing/events/create_events.rst

will autopopulate with pertinent data once attendees begin to register, booths and sponsors sign on
for the event, the event takes place, and so on.

Primarily, these smart buttons are used as logistical portals to perform specific actions for the
event. The numeric displays are primarily for quick reference points.

While those visual metrics are useful, they can still be clicked, and used to navigate to
specific event-related pages to modify and/or perform any desired actions.

Beneath the smart buttons is the event form, which contains various fields and clickable tabs that
serve to configure the initial, necessary details of the event.

The following are fields found on an event form:

- :guilabel:`Event Name`: the title of the event. This field is **required**.

  .. note::
     To the right of the entered :guilabel:`Event Name`, there is a language tooltip, represented by

>      an abbreviated language indicator (e.g. `EN`). When clicked, a :guilabel:`Translate:name`

Reviewer: Nitpick– add a space :)
```suggestion
     an abbreviated language indicator (e.g. `EN`). When clicked, a :guilabel:`Translate: name`
```

will autopopulate with pertinent data once attendees begin to register, booths and sponsors sign on
for the event, the event takes place, and so on.

> Primarily, these smart buttons are used as logistical portals to perform specific actions for the

Author: I fully understand (and appreciate) this suggestion. However, I fear that may lean into the "too basic" mentality we're trying to avoid these days. And, on top of that, this line is direct suggestion from ZST (from the PR referenced in the initial description), so I feel like this is the type of angle that I should be aiming for. So, all that said, I think I'm going to keep this particular section "as-is" -- but, if I get a similar note at a different review stage, I'll be sure to implement it (and give you FULL credit for pointing it out first) 👍

event. The numeric displays they showcase are merely secondary, and best used for quick reference
points.

While those visual metrics are useful, they can still be clicked, and used to transport the event
organizer to a specific event-related page, wherein they can modify and/or perform any desired
actions.

Beneath the smart buttons is the event form, which contains various fields and clickable tabs that
serve to configure the initial, necessary details of the event.

The following are fields found on an event form:

- :guilabel:`Event Name`: the title of the event. This field is **required**.

  .. note::
     To the right of the entered :guilabel:`Event Name`, there is a language tooltip, represented by
     an abbreviated language indicator (e.g. `EN`). When clicked, a :guilabel:`Translate:name`

>      pop-up window appears, from which various language translation options are presented, with each
>      option representing an already pre-configured language that is available in the database.

Reviewer 2: ```suggestion
     pop-up window appears, displaying various pre-configured language translation options available in the database.
```

  in Odoo. If this field is left blank, the event can be published on *all* websites in the
  database. To learn more, refer to the :doc:`Multiple websites
  <../../websites/website/configuration/multi_website>` documentation.

> - :guilabel:`Venue`: enter event venue details. This field pulls pertinent information
>   from the *Contacts* application.
>   Alternatively, :guilabel:`Venue` information can be manually added in this field, as well. At
>   the very least, there **must** be a venue name, address, city, zip code/region, and country
>   entered.

Reviewer: Nitpick– early line break
```suggestion
- :guilabel:`Venue`: enter event venue details. This field pulls pertinent information
  from the *Contacts* application. Alternatively, :guilabel:`Venue` information can be manually
  added in this field, as well. At the very least, there **must** be a venue name, address, city,
  zip code/region, and country entered.
```

  in Odoo. If this field is left blank, the event can be published on *all* websites in the
  database. To learn more, refer to the :doc:`Multiple websites
  <../../websites/website/configuration/multi_website>` documentation.

> - :guilabel:`Venue`: enter the pertinent event venue details in this field. This field relies on the
>   records present in the *Contacts* application, which pulls pertinent information into the form.
>   Alternatively, :guilabel:`Venue` information can be manually added in this field, as well. At
>   the very least, there **must** be a venue name, address, city, zip code/region, and country
>   entered.

Reviewer 2: ```suggestion
- :guilabel:`Venue`: enter event venue details. This field pulls pertinent information
  from the *Contacts* application.
  Alternatively, :guilabel:`Venue` information can be manually added in this field, as well. At
  the very least, there **must** be a venue name, address, city, zip code/region, and country
  entered.
```
